modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -14061,6 +14061,7 @@ regenerate-preview-and-thumbnail-of-video-files-in-documents-and-media=Regenerat
 regenerate-preview-and-thumbnail-of-video-files-in-documents-and-media-help=This action does not modify existing video previews.
 regenerate-preview-of-audio-files-in-documents-and-media=Regenerate preview of audio files in documents and media.
 regenerate-preview-of-audio-files-in-documents-and-media-help=This action does not modify existing audio previews.
+regeneration-actions=Regeneration Actions
```

Comment:
Not sure, but doesn't this sound weird? Maybe "Preview regeneration actions" would be better? I'm fine with sending this to Brian, though; he can probably make a better choice than us.